chrome-extension/vite.config.mts

```diff
@@ -20,10 +21,10 @@ export default defineConfig({
   plugins: [
     libAssetsPlugin({
       outputPath: outDir,
-    }),
-    watchPublicPlugin(),
-    makeManifestPlugin({ outDir }),
-    isDev && watchRebuildPlugin({ reload: true }),
+    }) as Plugin,
+    watchPublicPlugin() as Plugin,
+    makeManifestPlugin({ outDir }) as Plugin,
+    isDev && watchRebuildPlugin({ reload: true }) as Plugin,
```

Comment:
I think we can delete this assertions!

```suggestion
    watchPublicPlugin(),
    makeManifestPlugin({ outDir }),
    isDev && watchRebuildPlugin({ reload: true }),
```

Reply:
Without them I have errors during the build

```
│                                                   Type 'import("D:/dev/forks/chrome-extension-boilerplate-react-vite/node_modules/.pnpm/vite@5.4.1_@types+node@22.5.0_sass@1.77.8_terser@5.31.1/node_modules/
│                                                                                                                                                                                                           /
│ vite/dist/node/index").PluginOption' is not assignable to type 'import("D:/dev/forks/chrome-extension-boilerplate-react-vite/node_modules/.pnpm/vite@5.4.1_@types+node@20.14.10_sass@1.77.8_terser@5.31.1/nod
│                                                                                                                                                                                                           d
│ e_modules/vite/dist/node/index").PluginOption'.
│                                                     Type 'PluginOption[]' is not assignable to type 'PluginOption'.
│
│ 22     libAssetsPlugin({
│        ~~~~~~~~~~~~~~~~~
│ 23       outputPath: outDir,
│    ~~~~~~~~~~~~~~~~~~~~~~~~~
│ 24     }),
│    ~~~~~~
│
│   ../node_modules/.pnpm/vite@5.4.1_@types+node@20.14.10_sass@1.77.8_terser@5.31.1/node_modules/vite/dist/node/index.d.ts:3194:18
│     3194 declare function defineConfig(config: UserConfigExport): UserConfigExport;
│                           ~~~~~~~~~~~~
│     The last overload is declared here.
│
│
│ Found 1 error in vite.config.mts:22
│
│  ELIFECYCLE  Command failed with exit code 2.
```

Reply:
@govza That's unnecessary as @Jonghakseo said

The real issue is, the version of `@types/node` isn't specified for `chrome-extension` package, and this package(from screen) takes version of your machine(I'm also have node 22, but it's not working well for now)

![image](https://github.com/user-attachments/assets/1b634f7d-196a-4533-9712-83df4e61900c)

I will solve it in #697 and when you'll rebase it, this `as Plugins` wasn't longer necessary

Reply:
OO no, i'm wrong it's related to vite
![image](https://github.com/user-attachments/assets/738d0df3-080e-44ee-b81c-aa8f580e89b6)

Because of the lack of this package inside `chrome-extension` package, i need to fix it ASAP :)

Reply:
Yeah, that's it, it's simple, we have older version on the root package.json, and that's why this occur.

I'll open PR in the minutes.

This assertion isn't good solution.
We need to treat the cause, not the symptoms. 😄